cmd/serve/nfs/filesystem.go

```diff
@@ -0,0 +1,159 @@
+//go:build unix
```

Comment:
Is this necessary? Is it possible to run a NFS server on Windows, for example?

Reply:
Just cloned `willscott/go-nfs` to find out if there is any reference around it and these two are the only references to the word "build":

![image](https://github.com/rclone/rclone/assets/15693688/ab431a06-ba8d-4869-9918-0809655244d5)

Reply:
I remember I only added the `+build unix` after I saw failures in the CI. This needs to be tested on Windows, but that is beyond this PR.

Reply:
It would be a nice feature for this to work on Windows, but that can be worked on afterwards if you want to, or maybe @lucasew would like to.